machineconfiguration/v1alpha1/types_machineconfignode.go

```diff
@@ -13,6 +13,9 @@ import (
 // +openshift:api-approved.openshift.io=https://github.com/openshift/api/pull/1596
 // +openshift:file-pattern=cvoRunLevel=0000_80,operatorName=machine-config,operatorOrdering=01
 // +openshift:enable:FeatureGate=MachineConfigNodes
+// +kubebuilder:printcolumn:name="MCP",type="string",JSONPath=.spec.pool.name,priority=1
```

Comment:
Should this just be Pool Name? Not sure what others do but initialisms here seem a touch odd to me. Let's check a few other APIs for examples

Reply:
i dont mind chaning to `PoolName` 

when you say "initialisms" do you mean the jsonpath? i ran this and it pulled the specs just fine.